sopt/src/main/scala/dagr/sopt/cmdline/CommandLineException.scala

```diff
@@ -21,65 +21,66 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package dagr.core.cmdline
 
-private[core] object CommandLineException {
-  private[core] def format(msg: String, e: Exception): String = {
+package dagr.sopt.cmdline
+
+object CommandLineException {
+  def format(msg: String, e: Exception): String = {
     msg + ": " + e.getMessage + "\n" + e.getStackTrace.mkString("\n")
   }
 }
 
 /** Base class for all exceptions thrown by the command line parsing */
-private[core] class CommandLineException(msg: String) extends RuntimeException(msg) {
+class CommandLineException(msg: String) extends RuntimeException(msg) {
```

Comment:
Or actually, why not just chain to `RuntimeException`'s constructor that takes a prior?

```scala
class CommandLineException(msg: String, e: Option[Exception]) extends RuntimeException(msg, e getOrElse null)
```